src/core/features/user/pages/complete-profile/complete-profile.ts

```diff
@@ -0,0 +1,119 @@
+// (C) Copyright 2015 Moodle Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Component, OnDestroy } from '@angular/core';
+
+import { CoreSites } from '@services/sites';
+import { CoreDomUtils } from '@services/utils/dom';
+import { CoreLoginHelper } from '@features/login/services/login-helper';
+import { Translate } from '@singletons';
+import { CoreNavigator } from '@services/navigator';
+import { CoreEventObserver, CoreEvents } from '@singletons/events';
+import { CoreUtils } from '@services/utils/utils';
+
+/**
+ * Page that shows instructions to complete the profile.
+ */
+@Component({
+    selector: 'page-core-user-complete-profile',
+    templateUrl: 'complete-profile.html',
+})
+export class CoreUserCompleteProfilePage implements OnDestroy {
+
+    editingProfile = false;
+    logoutLabel: string;
+
+    protected urlLoadedObserver?: CoreEventObserver;
+    protected browserClosedObserver?: CoreEventObserver;
+
+    constructor() {
+        this.logoutLabel = CoreLoginHelper.getLogoutLabel();
+    }
+
+    /**
+     * Show a help modal.
+     */
+    showHelp(): void {
+        // @todo: Change this message in MOBILE-4059.
```

Comment:
```suggestion
        // @todo MOBILE-4059: Change this message.
```

It's better to follow this format to improve searchability.